tracer.go

```diff
@@ -266,3 +264,30 @@ func (t Tags) Apply(o *StartSpanOptions) {
 		o.Tags[k] = v
 	}
 }
+
+// Tag may be passed as a StartSpanOption to add a tag to new spans,
+// or its Set method may be used to apply the tag to an existing Span,
+// for example:
+//
+// tracer.StartSpan("opName", Tag{"Key", value})
+//
+//   or
+//
+// Tag{"key", value}.Set(span)
+type Tag struct {
+	Key   string
```

Comment:
nit: here you call it a `Key` (my pref, btw), elsewhere you call it a `Name`. Let's choose one and stick with it (key! key!).